### DISCUSSION TITLE
OMI glTF Working Group Meeting 2023-08-10

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/gltf-extensions#184] [General]
This meeting is on 08/10/2023 at 9:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda: 
- Let's talk physics PRs
- 

## Backlog:
* OMI Vehicles Extensions Research
* OMI / MX Components Extension
* We discussed https://openmv.org (Open Metaverse Foundation) and how we can synergize with their efforts. Should we make a showcase of our projects to open up a broader discussion?
* jin - proposals how to make our specs and work more accessible to others. showcase gltf models
* Review pull requests https://github.com/omigroup/gltf-extensions/pulls
* [EXT_skin_humanoid](https://github.com/takahirox/EXT_skin_humanoid)
* [OMI_link](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_link)
* [OMI_spawn_point](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_spawn_point)
* Review [MSFT_physics](https://github.com/eoineoineoin/glTF_Physics)
  * [glTF_Physics_Blender_Exporter](https://github.com/eoineoineoin/glTF_Physics_Blender_Exporter)
  * [glTF_Physics_Godot_Importer](https://github.com/eoineoineoin/glTF_Physics_Godot_Importer)
* Licensing Extension https://twitter.com/superhoge/status/1596757861882740736
* [KHR_audio / Khronos feedback](https://github.com/KhronosGroup/glTF/pull/2137)
  * [gltf-sample-assets](https://github.com/KhronosGroup/glTF-Sample-Models/pull/360)
* [Third Room Extensions](https://github.com/matrix-org/thirdroom/pull/469)
* [glXF](https://github.com/KhronosGroup/glXF/blob/main/specification/2.0/README.md)
* Show and tell
* Create next weeks agenda

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by aaronfranke** [comment]
Physics PRs https://github.com/omigroup/gltf-extensions/pull/169 and https://github.com/omigroup/gltf-extensions/pull/183 were discussed and merged during this meeting.

**Comment by madjin** [comment]
Vehicles can make use of OMI seat, Aaron proposed new extension for OMI_vehicle_wheel and OMI_vehicle_body (shown below is vehicle wheel)
![Screenshot from 2023-08-10 17-37-33](https://github.com/omigroup/gltf-extensions/assets/32600939/760bbd92-7674-42ba-973e-0130a84609be)

Briefly discussed artist workflows, which often involves blender
![Screenshot from 2023-08-10 17-51-10](https://github.com/omigroup/gltf-extensions/assets/32600939/00cd8b31-23a9-4829-9de4-2ed3e584c6d8)
![Screenshot from 2023-08-10 17-49-04](https://github.com/omigroup/gltf-extensions/assets/32600939/c97f3c06-9e38-4eed-8a06-3757a74f8456)

Lyuma helped me understand blender -> threejs/webgl platform vehicle workflow to fix wheels through parenting wheels to body, setting each wheel origin to geometry, and applying transforms for each. Here's more updated notes on interoperable vehicles, I added in some examples gathered from webgamedev discord: https://hackmd.io/@XR/interoperable-vehicles
